addons/medical/CfgEden.hpp

```diff
@@ -9,8 +9,8 @@ class Cfg3DEN {
             };
         };
         class GVAR(isMedicControl): Title {
-            attributeLoad = "(_this controlsGroupCtrl 100) lbsetcursel (((_value + 1) min 3) max 0);";
-            attributeSave = "(missionnamespace getvariable ['ace_isMedic_temp',0]) - 1;";
+            attributeLoad = "(_this controlsGroupCtrl 100) lbSetCurSel (((_value + 1) min 3) max 0);";
+            attributeSave = "(_this getVariable ['ace_isMedic_temp', 0]) - 1;";
```

Comment:
If we use a default value of 0 we can probably get away with just using the toolbox state, otherwise it'd always save to the SQM, albeit it'd be the correct value.